Kajabi/sage-lib
# Microphone icon - updated spelling for microphone icon reference

## Description

- [x] - updated spelling of the microphone icon

### Screenshots


|  before  |  after  |
|--------|--------|
|![Screen Shot 2020-12-07 at 3 25 15 PM](https://user-images.githubusercontent.com/1241836/101407712-ea31ff00-38a0-11eb-84d3-5bd178427c40.png)|![Screen Shot 2020-12-07 at 3 24 57 PM](https://user-images.githubusercontent.com/1241836/101407729-ef8f4980-38a0-11eb-9b0d-e8bc3549604f.png)|


## Test notes


### Steps for testing
1. Visit the icons page and verify that `microphone` is spelled correctly: http://localhost:4000/pages/design/icon
